kubernetes/resource_kubernetes_horizontal_pod_autoscaler_v2_test.go

```diff
@@ -0,0 +1,289 @@
+					resource.TestCheckResourceAttr("kubernetes_horizontal_pod_autoscaler.test", "spec.0.metric.#", "1"),
+					resource.TestCheckResourceAttr("kubernetes_horizontal_pod_autoscaler.test", "spec.0.metric.0.type", "External"),
+					resource.TestCheckResourceAttr("kubernetes_horizontal_pod_autoscaler.test", "spec.0.metric.0.external.0.metric.0.name", "latency"),
+					resource.TestCheckResourceAttr("kubernetes_horizontal_pod_autoscaler.test", "spec.0.metric.0.external.0.metric.0.selector.0.match_labels.lb_name", "test"),
+					resource.TestCheckResourceAttr("kubernetes_horizontal_pod_autoscaler.test", "spec.0.metric.0.external.0.target.0.type", "Value"),
+					resource.TestCheckResourceAttr("kubernetes_horizontal_pod_autoscaler.test", "spec.0.metric.0.external.0.target.0.value", "100"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccKubernetesHorizontalPodAutoscalerV2_importBasic(t *testing.T) {
+	resourceName := "kubernetes_horizontal_pod_autoscaler.test"
+	name := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckKubernetesHorizontalPodAutoscalerDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccKubernetesHorizontalPodAutoscalerV2Config_basic(name),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccCheckKubernetesHorizontalPodAutoscalerV2Destroy(s *terraform.State) error {
+	conn := testAccProvider.Meta().(*KubeClientsets).MainClientset
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "kubernetes_horizontal_pod_autoscaler" {
+			continue
+		}
+
+		namespace, name, err := idParts(rs.Primary.ID)
+		if err != nil {
+			return err
+		}
+
+		resp, err := conn.AutoscalingV2beta2().HorizontalPodAutoscalers(namespace).Get(name, metav1.GetOptions{})
+		if err == nil {
+			if resp.Namespace == namespace && resp.Name == name {
+				return fmt.Errorf("Horizontal Pod Autoscaler still exists: %s", rs.Primary.ID)
+			}
+		}
+	}
+
+	return nil
+}
+
+func testAccCheckKubernetesHorizontalPodAutoscalerV2Exists(n string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+
+		conn := testAccProvider.Meta().(*KubeClientsets).MainClientset
+
+		namespace, name, err := idParts(rs.Primary.ID)
+		if err != nil {
+			return err
+		}
+
+		_, err = conn.AutoscalingV2beta2().HorizontalPodAutoscalers(namespace).Get(name, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+
+		return nil
+	}
+}
+
+func testAccKubernetesHorizontalPodAutoscalerV2Config_basic(name string) string {
+	return fmt.Sprintf(`
+	resource "kubernetes_horizontal_pod_autoscaler" "test" {
+		metadata {
```

Comment:
These inlined HCL blocks should be formatted according to `terraform fmt`.
There's a nice tool to do that easily: https://github.com/katbyte/terrafmt

Reply:
Cool! Thanks